index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  </head>
+  <body>
+
+   <div class="app-wrap">
+     <header class="app-header">
+         <h1>Welcome</h1>
+         <figure>
+           <img src="images/Union.png" alt="union image">
+         </figure>
+     </header>
+     <div class="for_background">
+       <div class="headline">
```

Comment:
- [ ] Considering this `divs` are wrapping the exact same elements, you should consider removing one of them, to keep your code clean.
- [ ] Also, to keep the HTML semantic, you should consider making the remaining `<div>` a `<section>`. You can read more about sections [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).